wdl/ProcessNanoporeRNASamples.wdl

```diff
@@ -0,0 +1,77 @@
+version 1.0
+
+workflow ProcessNanoporeRNASamples {
+    input {
+        File bam
+        File bai
+        String platform
+        Boolean is_corrected
+    }
+
+    String docker_asm = "kgarimella/lr-asm:0.01.09"
```

Comment:
I think we can use the `struct RuntimeAttr` now, like the example in `AlignReads.wdl`

Reply:
I'll come by and ask you to show me how to do this; I'm not sure how to construct/populate structs in WDL 1.0.

Reply:
example 
`womtool validate MergeBams.wdl -i <EITHER_OF_THE_FOLLOWING_TWO>`

full specification
```json
{
  "merged_name": "whatever",
  "aligned_shards": ["gs://fdsfsd", "gs://dsfsdafsda"],

  "runtime_attr_override" : {"cpu_cores":2, "max_retries":1, "boot_disk_gb":10, "docker":"dockerdocker", "disk_gb":200, "mem_gb":16, "preemptible_tries":1}
}

```

partial specification
```json
{
  "merged_name": "whatever",
  "aligned_shards": ["gs://fdsfsd", "gs://dsfsdafsda"],

  "runtime_attr_override" : {"cpu_cores":2, "docker":"dockerdocker", "disk_gb":200, "mem_gb":16}
}
```